lib/hashie/extensions/coercion/active_model.rb

```diff
@@ -0,0 +1,85 @@
+require 'active_model/type'
+
+module Hashie
+  module Extensions
+    module Coercion
+      module ActiveModel
+        # Symbol is the one glaring type omission. Define it if it is not
```

Comment:
Yes, this comment was related to what's below. I find the comment less confusing having read the code, but really the dynamic class definition at runtime is my issue here.